backend/de.metas.business/src/main/java/de/metas/order/IOrderBL.java

```diff
@@ -122,15 +120,15 @@
 	/**
 	 * Set Target Sales Document Type.
 	 * This method is also setting IsSOTrx to true.
-	 *
-	 * @param soDocSubType sales DocSubType
 	 */
-	void setDocTypeTargetId(I_C_Order order, String soDocSubType);
+	void setDocTypeTargetId(I_C_Order order);
```

Comment:
please correct the javadoc.
Also, wdyt about renaming the metod to `setDefaultDocTypeTargetId()`